app/scripts/modules/core/src/presentation/main.less

```diff
@@ -945,14 +945,53 @@ select:invalid {
 }
 
 .section-heading-chevron {
-  transition: transform 0.15s ease;
-  margin-right: 4px;
+  &.left {
+    margin-right: 4px;
+  }
+
+  &.right {
+    margin-left: 4px;
+  }
+  display: flex;
 }
 
-.rotated-90 {
+@rotate-transition: transform 0.15s ease;
+
+.rotate-0 {
+  transition: @rotate-transition;
+  transform: rotate(0deg);
+}
+
+.rotate-m90 {
+  transition: @rotate-transition;
   transform: rotate(-90deg);
 }
 
+.rotate-p90 {
+  transition: @rotate-transition;
+  transform: rotate(90deg);
+}
```

Comment:
What does `m` and `p` stand for?

Reply:
Plus/minus. Is it better to rename to -rotate-90 for the negative classes?

Reply:
ah, I see that now.  What about `cw` and ccw`?  Is that more intuitive than `p` and `m`?

@caseyhebebrand @vigneshm @alanmquach